source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
```

Comment:
```suggestion
make use of :abbr:`Intel® Dynamic Platform and Thermal Framework (`Intel DPTF)`
```

Reply:
I see this was not packaged natively in Clear Linux because it is closed-source: https://github.com/clearlinux/distribution/issues/1073 

We should mention why it is not bundled and the limitations.